scripts/run_tests.sh

```diff
@@ -32,8 +32,14 @@ function display_result {
   fi
 }
 
-flake8 --max-complexity 10 --count
```

Comment:
Yeah looks like it https://pypi.python.org/pypi/pylint-mccabe although I didn't think about adding it so thanks for pointing it out